lab.R

```diff
@@ -151,32 +177,44 @@ focal_counties = c('Butte', 'Merced', 'Sacramento', 'Santa Clara')
 # google_mobility
 mob_df = read_csv(file.path('data', 'mobility.csv'))
 
-#' 1. *How many distinct values of `type` are there?  What does this variable indicate?*  
-#' 
-#' 
-#' 
+#' 1. *How many distinct values of `type` are there?  What does this variable indicate?* 
+skim(mob_df)
+#' The data has 11 types. "Type" indicates the different locations.
 
 #' 2. *How about `pct_diff`?* 
-#' 
+#' Integer percent change from baseline activity.
 #' 
 #' 
 
 #' 3. *During our time period of interest, does `mob_df` contain mobility data for every county in California?  If some counties are missing data, which ones are they?  Hints: There are 58 counties in California.  Try counting and then filtering to identify counties with outlying row counts.*  
-#' 
-#' 
-#' 
+length(unique(mob_df$county))
+mob_df %>% 
+    filter(is.na(pct_diff)) %>% 
+    summarise(length(unique(county)))
+# No, CA has 58 counties but this dataset only has 56 counties. There are 28 counties with missing data. 
 
 #' 4. *In the `plots` folder, take a look at `mobility.png`.  Recreate this plot.  (Use whatever theme and colors that you like.  To create a horizontal line: `geom_hline(yintercept = 0, alpha = .5)`.  You don't need to save to disk.)* 
-#' 
+df_plot = mob_df %>% 
+    filter(county == "Butte"|
+               county == "Merced"|
+               county =="Sacramento"|
+               county == "Santa Clara") %>% 
```

Comment:
You can also use `%in%` to simplify the test:  
`county %in% c("Butte", "Merced", "Sacramento", "Santa Clara")`